easybuild/easyconfigs/c/Cufflinks/Cufflinks-2.2.1-foss-2015b.eb

```diff
@@ -0,0 +1,28 @@
+name = 'Cufflinks'
+version = '2.2.1'
+
+homepage = 'http://cole-trapnell-lab.github.io/cufflinks/'
+description = """Transcript assembly, differential expression, and differential regulation for RNA-Seq"""
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+toolchainopts = {'optarch': True, 'pic': True}
+
+source_urls = ['http://cole-trapnell-lab.github.io/cufflinks/assets/downloads/']
+sources = [SOURCELOWER_TAR_GZ]
+
+dependencies = [
+    ('Boost', '1.60.0', '-Python-2.7.10'),
+    ('SAMtools', '0.1.20'),
+    ('Eigen', '3.2.7'),
+    ('zlib', '1.2.8'),
+]
+
+preconfigopts = 'CPPFLAGS=-I$EBROOTEIGEN/include'
+configopts = '--with-boost=$EBROOTBOOST --with-bam=$EBROOTSAMTOOLS'
```

Comment:
existing easyconfig for Cufflinks v2.2.1 uses ` --with-bam-libdir=${EBROOTSAMTOOLS}/lib`?